### PR TITLE
add notice text as #2216 third suggestion

### PR DESCRIPTION
**Pull request by ppKrauss**

### CODE REVIEW

**Comment by osqueryer** [comment]
Can one of the osquery admins reply with "ok to test" to kick off a build...

**Comment by facebook-github-bot-0** [comment]
Thank you for your pull request and welcome to our community. We require contributors to sign our Contributor License Agreement, and we don't seem to have you on file. In order for us to review and merge your code, please sign up at https://code.facebook.com/cla - and if you have received this in error or have any questions, please drop us a line at cla@fb.com. Thanks!

File: docs/wiki/introduction/sql.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ The osquery SQL language is a superset of SQLite's, please read [SQL as understo
 
 `SELECT` only! All mutation-based verbs exist, like `INSERT`, `UPDATE`, `DELETE`, and `ALTER` but they do not do anything-- except if you're fancy and creating run-time tables or `VIEW`s. ;)
 
+> NOTICE: several tables, `file` for example, requires a predicate for one of the columns, and **will not work without it**. See  [Tables with arguments](#tables-with-arguments) for more information.
```
Comment (theopolis) [inline docs/wiki/introduction/sql.md]: Several should be capitalized.

s/requires/require/

File: docs/wiki/introduction/sql.md [hunk 2]
```diff
@@ -123,7 +125,7 @@ We can expand upon this later using subqueries and more tables.
 
 ### Tables with arguments
 
-Several tables, `file` for example, represent concepts that require arguments. Consider `SELECT * FROM file`, you do not want this to trigger a complete walk of the mounted file systems. It is an ambiguous concept without some sort of argument or input parameter. These tables, and their columns, are flagged in the [table documentation](https://osquery.io/docs/tables/) as requiring a column or as using a column to generate additional information.
+Several tables, `file` for example, represent concepts that require arguments. Consider `SELECT * FROM file`, you do not want this to trigger a complete walk of the mounted file systems. It is an ambiguous concept without some sort of argument or input parameter. These tables, and their columns, are flagged by a *dropper icon* in the [table documentation](https://osquery.io/docs/tables/)  as requiring a column or as using a column to generate additional information.
```
Comment (theopolis) [inline docs/wiki/introduction/sql.md]: Is there a reason another " " is added after the link here?

Comment (ppKrauss) [inline docs/wiki/introduction/sql.md]: No, but no problem in a md file. I can fix it

**Comment by theopolis** [comment]
@ppKrauss you can continue to edit I believe, if you're editing via the Github webpage it should give you an option to add your change to your auto-generated `patch-1` branch.

**Comment by theopolis** [comment]
screen-shot-2225

File: docs/wiki/introduction/sql.md [hunk 3]
```diff
@@ -12,6 +12,8 @@ The osquery SQL language is a superset of SQLite's, please read [SQL as understo
 
 `SELECT` only! All mutation-based verbs exist, like `INSERT`, `UPDATE`, `DELETE`, and `ALTER` but they do not do anything-- except if you're fancy and creating run-time tables or `VIEW`s. ;)
 
+> NOTICE: Several tables, `file` for example, requires a predicate for one of the columns, and **will not work without it**. See  [Tables with arguments](#tables-with-arguments) for more information.
```
Comment (zwass) [inline docs/wiki/introduction/sql.md]: Sorry to nitpick, but "requires" should be "require", and there is an extra space after "See".

Comment (ppKrauss) [inline docs/wiki/introduction/sql.md]: Ops, thanks, correctiong in a minute...

**Pull request merged**